packages/styled-components/src/constructors/constructWithOptions.ts

```diff
@@ -8,7 +8,7 @@ import {
   Runtime,
   StyledOptions,
   StyledTarget,
-  Styles,
+  Styles
```

Comment:
Not sure why you removed this comma. When I run prettier with the provided `.prettierrc`, it keeps the trailing commas in imports.